README.md

```diff
@@ -1,34 +1,52 @@
 
-This repo tries to collect ideas for Open Source projects that are up for grabs. It's much better for a project to see the light of day instead of slowly dying in a dusty Reminder-list in some way too small iCloud storage.
+This repo tries to collect ideas for Open Source projects that are up for grabs. It's much better for a project to see the light of day instead of slowly dying in a dusty Reminders-list in some way-too-small iCloud storage.
 
-**Let's show what open in Open Source means, by creating a welcoming, inclusive and supporting community for all that want to be involved.** My hope is that we can get more people involved with Open Source, learning the customary behavior and to continue to drive the open community forward. [Read more about the initiative in the blog post](https://hackernoon.com/open-source-ideas-initiative-ca747121ac34).
+**Let's show what open in Open Source means by creating a welcoming, inclusive and supportive community for all that want to be involved.** My hope is that we can get more people involved with Open Source, learning the customary behavior and continuing to drive the open community forward. [Read more about the initiative in the blog post](https://hackernoon.com/open-source-ideas-initiative-ca747121ac34).
 
 <div align="center">
 <img src="https://github.com/open-source-ideas/open-source-ideas/blob/master/collaboration-five.jpg?raw=true" alt="Epic Collaboration Five" />
 </div>
 
 ---
 
-See all the [current project suggestions](https://github.com/open-source-ideas/open-source-ideas/issues). Or filter on
+See all the [current project suggestions](https://github.com/open-source-ideas/open-source-ideas/issues). Or filter by required time (ETA):
 
 * [`Little work`](https://github.com/open-source-ideas/open-source-ideas/labels/Little%20work)
 * [`Medium work`](https://github.com/open-source-ideas/open-source-ideas/labels/Medium%20work)
 * [`Much work`](https://github.com/open-source-ideas/open-source-ideas/labels/Much%20work)
 
-Or by estimated required level:
+Or filter by estimated experience level:
 
 * [`Beginner`](https://github.com/open-source-ideas/open-source-ideas/issues?q=is%3Aissue+is%3Aopen+label%3ABeginner)
 * [`Intermediate`](https://github.com/open-source-ideas/open-source-ideas/issues?q=is%3Aissue+is%3Aopen+label%3AIntermediate)
 * [`Advanced`](https://github.com/open-source-ideas/open-source-ideas/issues?q=is%3Aissue+is%3Aopen+label%3AAdvanced)
 
-Or if you are totally new to open source and development, you can check out
+Or filter by tech category:
+
+* [`Mobile app`](https://github.com/open-source-ideas/open-source-ideas/labels)
```

Comment:
The most of categories suggested are IMHO irrelevwnt. IMHO they should be related to the special skills needed. Not `IoT`,  but `hardware design`. Not `Bots` and `Security`, but `reverse engineering`. See https://github.com/open-source-ideas/meta/issues/6

Reply:
Thanks for the input. Hmm, a question I would pose here is:
Do you think someone (software engineer or a brand-new developer) looking at the open Issues for cool software-based weekend projects that excite them would search by the label `electric engineering` or `reverse engineering`? These are too broad in my opinion. When someone sees `Bots` they think, "Wow, I've always wanted to built a chatbot for Slack"; they immediately know what it is. But if they see `reverse engineering` would they know what this actually entails?

Reply:
>These are too broad in my opinion. 

They reflect peoples skills, tools and professional interests. And labels are meant to be broad, narrow topics are searcheable by keywords.

>When someone sees Bots they think, "Wow, I've always wanted to built a chatbot for Slack"; they immediately know what it is.

I don't think so. `Bots` is a of end products, and it is a both a too narrow one to be useless for the ones interested in AI  and data science in general (I guess noone interested in AI is mad enough to start bruteforcing the search by inserting random words reoated to AI) and is a too broad one for the ones interested in one specific bot doing one thing needed for them. 

`AI` is a broad enough category, so the ones interested in AI would just filter by it to find the projects potentially matching their skills.

Reply:
Hmm, I want to lay out my point via a scenario involving a series of steps (my thoughts in parenthesis):
1. Someone (let's call her Mary) comes to "open-source-ideas" because they want to work on something this weekend
2. Mary has always been interested in chatbots and wants hands-on experience building them (I think we can agree this is a valid scenario, someone wants hands-on experience working on something specific that interests them)
3. Mary navigates to the open issues. She sees 100+ issues (a lot). She wants to filter down her search to match her interests.
4. She sees that Labels can be applied! She sees the `reverse engineering` label. She knows this includes chatbots (I don't see this happening).

I don't see the above scenario occurring. In my opinion, `Bots` is just the right amount of specificity for 2 reasons: 1) you know it involves chatbots, plain and simple 2) this could be Facebook Messenger bots, Slack bots, Twilio bots, Alexa bots, etc. Specific enough to allow Mary to find what interests her quickly, and broad enough to include a category of different items. I personally don't see Mary having any idea that in her case, she should filter down by `reverse engineering` to find what she is looking for.

Reply:
>Mary has always been interested in chatbots

Then he searches by the word "chatbot"

>She sees the reverse engineering label. She knows this includes chatbots (I don't see this happening).

`reverse engineering` doesn't includes chatbots (unless one wants to reverse engineer some existing "smart loudspeaker", of course), it includes bots for online games.

Reply:
In addition, I feel what is also being underrated at this time is the **motivation/excitement** factor. I think it's difficult getting a random person to complete a random idea shared by another random person. They need to be excited about it and want to finish it because it's so cool! This involves grabbing their attention. Putting the _utility_ of labels aside for a moment, would you agree that having colorful, cool, informative labels with `AI/ML`, `Mobile app`, etc. give someone who doesn't know what they are looking for excitement that they can work with these cool technologies? Sure, they can keyword search it through the filter text bar, but does that have the same effect? It seems the spirit of the project is to get these projects _done_ so we can all benefit, and I still feel this could contribute towards this goal.

Reply:
Yeah, IoT is not searcheable. I cannot explain why I don't think IoT is a good label. I can imagine the situation people searching for `IoT` in this repo, for example [bulk analysis of firmware of IoT](http://firmware.re/), but I don't think this cases are frequent enough. I am sorry. I may be wrong.


So I suggest to do the experiment. Add all the possible labels, mark all the issues with all the labels that match, wait some time, then do PCA and exclude some labels highly correlated with other ones. Then remove all the labels used not frequently enough.

Reply:
Thanks for your input! That's a smart approach.